src/navability/services/Utils.jl

```diff
@@ -31,4 +31,14 @@ function waitForCompletion(
             wait_time <= 0 && throw(error(exceptionMessage))
         end
     end
+end
+
+# Dispatch for vector of Tasks
+function waitForCompletion(
+    navAbilityClient::NavAbilityClient,
+    requestIds::AbstractVector{<:Task};
```

Comment:
I need to try this, curious how the task thing works

Reply:
In Python when you do `async def` you get a `<coroutine object...>` back.  In Julia `@async` gives a `Task` object back.  Its the same thing.  The Julia one is also fully consolidated with mutlithreaded tasks via dispatch.